eval/compile.rkt

```diff
@@ -270,8 +270,10 @@
   (define registers (make-vector register-count))
   (define repeats (make-vector register-count #f)) ; flags whether an op should be evaluated
   (define precisions (make-vector register-count)) ; vector that stores working precisions
-  ;; starting precisions for the first, un-tuned iteration
-  (define initial-precisions (setup-vstart-precs instructions (length vars) roots discs))
+  ;; vector for adjusting precisions
+  (define incremental-precisions (setup-vstart-precs instructions (length vars) roots discs))
```

Comment:
`incremental-precisions` stores the _traversed_ precisions with "+ 5" precision addition for every other node.
This vector is to be used in `backward-pass`